Security Recommendations/Private preview recommendations/TVM-recommendations/SecurityPolicies/TVM-EnableGuestExtension-Audit/azurepolicy.json

```diff
@@ -0,0 +1,99 @@
+{
+  "properties": {
+    "displayName": "Install an Attestation extension on your virtual machines (preview)",
+    "policyType": "Custom",
+    "mode": "All",
+    "description": "Your Virtual Machine supports Measured Boot. Install the Attestation extension to allow Azure Security Center to proactively attest and monitor your virtual machine�s boot integrity.",
+    "metadata": {
+      "securityCenter": {
+        "remediationDescription": "Step 1 --> Step 2 --> Step 3 --> Step 4",
+        "severity": "Low"
+      },
+      "version": "1.0.0-preview",
+      "category": "Compute",
+      "preview": true
+    },
+    "parameters": {
+      "effect": {
+        "type": "string",
+        "defaultValue": "AuditIfNotExists",
+        "allowedValues": [
+          "AuditIfNotExists",
+          "Disabled"
+        ],
+        "metadata": {
+          "displayName": "Effect",
+          "description": "Enable or disable the execution of the policy"
+        }
+      }
+    },
+    "policyRule": {
+      "if": {
+        "allOf": [
+          {
+            "field": "type",
+            "equals": "Microsoft.Compute/virtualMachines"
+          },
+          {
+            "field": "Microsoft.Compute/virtualMachines/storageProfile.imageReference.offer",
+            "in": [
+              "windowsserver-gen2preview-preview",
+              "0003-com-ubuntu-server-trusted-vm"
+            ]
+          },
+          {
+            "field": "Microsoft.Compute/virtualMachines/storageProfile.imageReference.sku",
+            "in": [
+              "18_04-gen2",
+              "windows10-tvm",
+              "windowsserver2019-tvm",
+              "windowsserver2016-tvm"
+            ]
+          },
+          {
+            "field": "Microsoft.Compute/virtualMachines/securityProfile",
+            "exists": "true"
+          },
+          {
+            "field": "Microsoft.Compute/virtualMachines/securityProfile.enableVTPM",
+            "equals": "true"
+          }
+        ]
+      },
+      "then": {
+        "effect": "[parameters('effect')]",
+        "details": {
+          "type": "Microsoft.Compute/virtualMachines/extensions",
+          "existenceCondition": {
+            "allOf": [
+              {
+                "field": "Microsoft.Compute/virtualMachines/extensions/publisher",
+                "in": [
+                  "Microsoft.Azure.Security",
+                  "Microsoft.Azure.Security.WindowsAttestation",
+                  "Microsoft.Azure.Security.LinuxAttestation"
+                ]
+              },
+              {
+                "field": "Microsoft.Compute/virtualMachines/extensions/type",
+                "in": [
+                  "GuestAttestation"
+                ]
+              },
+              {
```

Comment:
does it check if attestation extension is successfully provisioned ? Or it will pass if some other extension is provisioned ?